docs/connector-development/README.md

```diff
@@ -1,96 +1,108 @@
-
-If you need help with connector development, we offer premium support to our open-source users,
-[talk to our team](https://airbyte.com/talk-to-sales-premium-support) to get access to it.
+If you need to build a connector for an API Source, start with Connector Builder. It'll be enough
+for most use cases. If you need help with connector development, we offer premium support to our
+open-source users, [talk to our team](https://airbyte.com/talk-to-sales-premium-support) to get
+access to it.
 
 ### Connector Builder
 
-The [connector builder UI](connector-builder-ui/overview.md) is based on the low-code development framework below and
-allows to develop and use connectors without leaving the Airbyte web UI. No local developer environment required.
+The [connector builder UI](connector-builder-ui/overview.md) is based on the low-code development
+framework below and allows to develop and use connectors without leaving the Airbyte web UI. No
+local developer environment required.
 
 ### Low-code Connector-Development Framework
 
-You can use the [low-code framework](config-based/low-code-cdk-overview.md) to build source connectors for HTTP API
-sources. Low-code CDK is a declarative framework that provides a YAML schema to describe your connector without writing
-any Python code, but allowing you to use custom Python code if required.
-
+You can use the [low-code framework](config-based/low-code-cdk-overview.md) to build source
+connectors for HTTP API sources. Low-code CDK is a declarative framework that provides a YAML schema
+to describe your connector without writing any Python code, but allowing you to use custom Python
+components if required.
 
 ### Python Connector-Development Kit \(CDK\)
 
-You can build a connector in Python with the [Airbyte CDK](cdk-python/). Compared to the low-code CDK, the Python CDK is
-more flexible, but building the connector will be more involved. It provides classes that work out of the box for most
-scenarios, and Airbyte provides generators that make the connector scaffolds for you.
+You can build a connector in Python with the [Airbyte CDK](cdk-python/). Compared to the low-code
+CDK, the Python CDK is more flexible, but building the connector will be more involved. It provides
+classes that work out of the box for most scenarios, and Airbyte provides generators that make the
+connector scaffolds for you. Here's a guide for
+[building a connector in Python CDK](tutorials/custom-python-connector/0-getting-started.md).
 
 ### Community maintained CDKs
 
 The Airbyte community also maintains some CDKs:
 
-- The [Typescript CDK](https://github.com/faros-ai/airbyte-connectors) is actively maintained by Faros.ai for use in their product.
+- The [Typescript CDK](https://github.com/faros-ai/airbyte-connectors) is actively maintained by
+  Faros.ai for use in their product.
 - The [Airbyte Dotnet CDK](https://github.com/mrhamburg/airbyte.cdk.dotnet) in C#.
```

Comment:
that dotnet cdk hasn't been update in 3 years. might be worth deleting

Reply:
I had the same thought. But thought okay, I'll leave it there for now.